### PR TITLE
fix: make sure all PHP extensions required by PHPUnit are installed

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: .github/composite-actions/setup-php/action.yml [hunk 1]
```diff
@@ -22,4 +22,4 @@ runs:
         php-version: ${{ inputs.version }}
         coverage: ${{ inputs.coverage }}
         tools: composer, ${{ inputs.tools }}
-        extensions: none, curl, dom, json, mbstring, opcache, openssl, simplexml, tokenizer, xml, xmlwriter, zip
+        extensions: none, ctype, curl, dom, json, mbstring, opcache, openssl, simplexml, tokenizer, xml, xmlwriter, zip
```
Comment (mvorisek) [inline .github/composite-actions/setup-php/action.yml:25]:
```suggestion
        extensions: ctype, curl, dom, json, mbstring, opcache, openssl, simplexml, tokenizer, xml, xmlwriter, zip
```

;-)

Comment (Wirone) [reply .github/composite-actions/setup-php/action.yml:25]: As far as I remember it is a trick to start from scratch and install only listed extensions. Without `none` also other extensions are installed, even these we don't need.

Comment (kubawerlos) [reply .github/composite-actions/setup-php/action.yml:25]: It's exactly how @Wirone explained, it is quite old test here: https://github.com/werlos/github-actions-php-extensions - but shows the difference it makes.

**Review by Wirone — APPROVED** [review]

**Comment by Wirone** [comment]
Thank you very much @kubawerlos 🍻

**Pull request merged**